modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -257,6 +266,71 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-169960. Verify that a submited MDF will be displayed on the list"
+	@priority = 5
+	test CanViewMDFStatusOnListing {
+		task ("Given I go to add a new MDF Request and when I fill all the goals") {
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Test Description",
+				campaignName = "First Campaign",
+				countryName = "Global",
+				goalsOptions = "Thought leadership",
+				targetMarketOptions = "Aerospace & Defense,Energy",
+				targetRolesOptions = "Director/Manager");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And When I add a new activity") {
+			PRMMDFRequest.newActivity(
+				activityDescription = "Activity Description",
+				activityName = "Activity Name 1",
+				budgetList = "Sponsorship Fee-2000,Room Rental-3000",
+				followUpStrategy = "test",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetOfLeads = "test",
+				typeOfActivity = "Miscellaneous Marketing");
+
+			Button.click(button = "Continue");
+
+			Button.click(button = "Continue");
+
+			Button.click(button = "Submit");
+		}
+
+		task ("Then the MDF status appears on list") {
+			PRMMDFListing.viewOnList(
+				mdfStatus = "Pending Marketing Review",
+				rowNumber = "Status");
+		}
+
+		task ("And when I change the MDF Request status") {
+			PRMMDFRequest.changeMDFStatus(
+				campaignName = "First Campaign",
+				key_statusButton = "Approve");
+
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "/home",
+				buttonText = "Home");
+		}
+
+		task ("Then the status of a MDF Request was change on list") {
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "mdf-requests",
+				buttonText = "MDF");
```

Comment:
Hi @mateussandes23, this code should be inside another task, that could be 'And when I go back to the MDF Listing Page'